### PR TITLE
Project README and scaffold

### PR DESCRIPTION
**Pull request by Hamled**
# Description
This PR includes the complete project README, the feedback form and PR template, as well as the project scaffold.

As explained in the README, the scaffold for this project is largely the baseline from `create-react-app`, except with CSS styles and the JSON data already provided.

# Things to look at
* Are the baseline instructions too much?
* Are the instructions for each wave too sparse? I tried to keep them simple and avoid mentioning any specific components.

# Concerns
I have a concern that there will be too much effort required of the students to "reverse engineer" my HTML structure based on the CSS classes provided.

I included explicit instructions for students to ask us for help if they're having difficulty or confusion on CSS issues, but this still might be too much.

### CODE REVIEW

**Comment by Hamled** [comment]
[React Chat Log project](https://trello.com/c/ZggcmQ3S/868-react-chat-log-project)

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# React Timeline
```
Comment (tildeee) [inline .github/pull_request_template.md]: React Chatlog!

Comment (tildeee) [inline .github/pull_request_template.md]: React Chatlog!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Congratulations! You're submitting your assignment!
+
+## Comprehension Questions
+Question | Answer
+:------------- | :-------------
+Explain what purpose a component serves in React? |
```
Comment (tildeee) [inline .github/pull_request_template.md]: Not a question?

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
+/**************/
+/* App styles */
+/**************/
```
Comment (tildeee) [inline src/App.css]: Admittedly, in the styles lecture (PR #422 in textbook-curriculum), I mention that every component has its own .css file lololol... we should either acknowledge this in this project's readme that we have it all in one file (maybe even acknowledge that they can change this CSS), or we should change the styles lecture (I'm happy to do that)

**Comment by Hamled** [comment]
@tildeee Okay, I'm glad to have a confirmation that the styling instructions are appropriate.

I've added a bit that makes it clear that having all the styles in `src/App.css` is not our suggested convention, and is only being done for this project's scaffold.

Also, given that they will only have a single afternoon to work on this, I think it would be better to say that if they're struggling for half an hour with CSS they should come ask us.

File: README.md [hunk 4]
```diff
@@ -1 +1,63 @@
...
+
+## Wave 1
+For wave 1 implement the component(s) necessary to display a single chat message bubble with the message text and relative timestamp, plus the sender's name above it.
+
+A good way to test this code as you write it would be to take the first chat message from the JSON data file and use it as the data for the component(s) you are implementing.
+
+### Styling
+The styles necessary to make an app which visually matches the demo above have been provided for you, but you will need to make use of them in your React components by adding classes to specific HTML elements in your JSX.
+
+There are several CSS classes for the different elements of a chat message. You should attempt to determine which classes should be applied to which HTML elemnts, however if you are stuck on this please ask a member of the instructional staff as the styling on this project is not the learning goal.
```
Comment (tildeee) [inline README.md]: "HTML elemnts" -> "HTML elements"

**Comment by tildeee** [comment]
Just noticed that the casing of `TimeStamp` the component doesn't match the filename `Timestamp.js`

**Pull request merged**